Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragmentTest.kt

```diff
@@ -18,6 +26,25 @@ import testhelpers.takeScreenshot
 @RunWith(AndroidJUnit4::class)
 class OnboardingFragmentTest : BaseUITest() {
 
+    @MockK lateinit var appConfigProvider: AppConfigProvider
+
+    @Before
+    fun setup() {
+        MockKAnnotations.init(this)
+
+        every { appConfigProvider.currentConfig } returns flowOf(
+            mockk<ConfigData>().apply {
+                every { maxEncounterAgeInDays } returns 14
```

Comment:
The new value is '10', right? Let's use this one to have the current value in the screenshots?

Reply:
I think we should use design documents from Figma as the source of truth for screenshots. The value there is 14, but I double-check with designers.

Reply:
will update it in other PR if it's necessary